packages/fgtclb/academic-persons/Classes/Command/CreateProfilesCommand.php

```diff
@@ -31,12 +32,32 @@ public function __construct(private readonly FrontendUserProvider $frontendUserP
 
     protected function configure(): void
     {
-        $this->setHelp('This command create profiles for all frontend users that do not have a profile yet but should have one.');
+        $this
+            ->setHelp('This command create profiles for all frontend users that do not have a profile yet but should have one.')
+            ->addOption(
+                'exclude-pids',
+                'e',
+                InputArgument::OPTIONAL,
```

Comment:
Mixing `InputArgument` for `addOption()` is wrong.

This should be `InputOption::VALUE_REQUIRED`, which requires that a value is provided when the option is added - still having the option optional.

Same for the include-pids option.